tools/vol2obj/main.c

```diff
@@ -110,21 +241,7 @@ static char _output_mesh_filename[MAX_FILENAME_LEN]; // e.g. `output_frame_00000
 static char _output_mtl_filename[MAX_FILENAME_LEN];  // e.g. `output_frame_00000000.mtl`
 static char _output_img_filename[MAX_FILENAME_LEN];  // e.g. `output_frame_00000000.jpg`
 static char _material_name[MAX_FILENAME_LEN];        // e.g. `volograms_mtl_00000000`
-
-/// Globals for parsing the command line arguments in a function.
-static int my_argc;
-static char** my_argv;
-
-/** Look for a string amongst command-line arguments.
- * @param check_str The string to match (case insensitive).
- * @return If not found returns 0. Otherwise returns the index number into argv matching `check_str`.
- */
-static int _check_param( const char* check_str ) {
-  for ( int i = 1; i < my_argc; i++ ) {
-    if ( !strcasecmp( check_str, my_argv[i] ) ) { return i; }
-  }
-  return 0;
-}
+static char _prefix_str[MAX_FILENAME_LEN];           // defaults to `output_frame_`
```

Comment:
the original point of the PR was just to add this one string in as a new command line option. `--prefix mysequence_`